From: Jiri Slaby <jslaby@suse.cz>
Date: Thu, 23 Jan 2014 02:04:59 +0100
Subject: kgr: trigger the first check earlier
Patch-mainline: submitted for review
References: fate#313296

In 10 seconds, not 30. This speeds up the whole process in most
scenarios.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
Cc: Steven Rostedt <rostedt@goodmis.org>
Cc: Frederic Weisbecker <fweisbec@gmail.com>
Cc: Ingo Molnar <mingo@redhat.com>
---
 kernel/kgraft.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/kgraft.c b/kernel/kgraft.c
index 9d742d04af1d..e805c2f76a68 100644
--- a/kernel/kgraft.c
+++ b/kernel/kgraft.c
@@ -329,7 +329,7 @@ int kgr_patch_kernel(struct kgr_patch *patch)
 	/*
 	 * give everyone time to exit kernel, and check after a while
 	 */
-	queue_delayed_work(kgr_wq, &kgr_work, KGR_TIMEOUT * HZ);
+	queue_delayed_work(kgr_wq, &kgr_work, 10 * HZ);
 
 	return 0;
 err_unlock:
-- 
2.0.1

